torchvision/prototype/transforms/_geometry.py

```diff
@@ -234,7 +234,18 @@ def _check_inputs(self, flat_inputs: List[Any]) -> None:
 
     def _transform(
         self, inpt: Union[datapoints.ImageType, datapoints.VideoType], params: Dict[str, Any]
-    ) -> Union[List[datapoints.ImageTypeJIT], List[datapoints.VideoTypeJIT]]:
+    ) -> Tuple[
```

Comment:
Yes, all of this is ugly and verbose AF. We can also try to keep the `List` annotation (which we should have gone for in the first place).